pyaedt/modules/SetupTemplates.py

```diff
@@ -1965,9 +2016,7 @@ def add_subrange(self, type, start, end=None, count=None, unit="GHz", clear=Fals
                 self.props["RangeSamples"] = count
             self.props["SweepRanges"] = {"Subrange": []}
             return self.update()
-        range = {}
-        range["RangeType"] = type
-        range["RangeStart"] = str(start) + unit
+        range = {"RangeType": type, "RangeStart": str(start) + unit}
```

Comment:
@gmalinve If possible, can you rename the `range` variable because it is a built-in function and whenever possible we want to avoid modifying the behavior of those functions. 
Maybe `subrange`.

Reply:
sure! :)